src/index.js

```diff
@@ -0,0 +1,65 @@
+
+/**
+ * Module dependencies.
+ */
+
+import element from 'virtual-element';
+import Clipboard from 'clipboard';
+import uid from 'uid';
+
+/**
+ * Exports.
+ */
+
+export default {
+  initialState,
+  afterMount,
+  render
+};
+
+/**
+ * Initial state.
+ */
+
+function initialState() {
+  return {
+    copied: false
+  };
+}
+
+/**
+ * After mount.
+ */
+
+function afterMount({ props }, el, setState) {
+  const id = el.id = el.id || `x${uid(30)}`;
```

Comment:
this was a hack since `uid` can generate invalid ids.  maybe use [`random-id`](https://www.npmjs.com/package/random-id)?